package/MDAnalysis/topology/PDBParser.py

```diff
@@ -515,3 +515,94 @@ def _parse_conect(conect):
                   range(n_bond_atoms))
     return atom_id, bond_atoms
+
+def fetch_pdb(PDB_IDS=None,
+            cache_path=None,
+            progressbar=False,
+            file_format="pdb.gz",
+            ):
+    
+    """
+    Download one or more PDB files from the RCSB Protein Data Bank and cache them locally.
+
+    Given one or multiple PDB IDs, downloads the corresponding structure files in the specified
+    format and stores them in a local cache directory. If files are cached on disk, fetch_pdb() will skip the download and use
+    the cached version instead.
+    
+    Returns the path(s) as a string to the downloaded files.
+
+    Parameters
+    ----------
+    PDB_IDS : str or sequence of str
+        A single PDB ID as a string, or a sequence of PDB IDs to fetch.
+    cache_path : str or pathlib.Path
+        Directory where downloaded file(s) will be cached. 
+    file_format : str
+        The file extension/format to download (e.g., "cif", "pdb")
+    progressbar : bool, optional
+        If True, display a progress bar during file downloads. Default is False.
+
+    Returns
+    -------
+    str or list of str
+        The path(s) to the downloaded file(s). Returns a single string if one PDB ID is given,
+        or a list of strings if multiple PDB IDs are provided.
+
+    Raises
+    ------
+    requests.exceptions.HTTPError
+        If an invalid PDB code or file format is specified.
+    
+    Notes
+    -----
+    This function downloads using the API established here at https://www.rcsb.org/docs/programmatic-access/file-download-services.
+
+    Examples
+    --------
+    Download a single PDB file:
+
+    >>> mda.fetch_pdb("1AKE", file_format="cif")
+    './pdb_cache/1AKE.cif'
+
+    Download multiple PDB files with a progress bar:
+
+    >>> mda.fetch_pdb(["1AKE", "4BWZ"], progressbar=True)
+    ['./pdb_cache/1AKE.pdb.gz', './pdb_cache/4BWZ.pdb.gz']
+
+    Download a single PDB file and converting it to a universe:
+
+    >>> mda.Universe(mda.fetch_pdb("1AKE"), file_format="pdb.gz")
+    <Universe with 3816 atoms>
+
+    Download multiple PDB files and converting them to a universe:
+
+    >>> [mda.Universe(mda.fetch_pdb(PDB_ID), file_format="pdb.gz") for PDB_ID in ("1AKE", "4BWZ")]
+    [<Universe with 3816 atoms>, <Universe with 2824 atoms>]
+
+    """
+    
+    # Have to do this dictionary approach instead of using Pooch.retrieve in order to prevent the hardcoded known_hash warning from showing up
+    
+    try:
+        import pooch
+    except:
+        raise ModuleNotFoundError('pooch is needed as a dependency for fetch_pdb()')
+
+    if isinstance(PDB_IDS, str):
+        PDB_IDS = (PDB_IDS,)
+
+    if cache_path is None:
+        cache_path = pooch.os_cache('MDAnalysis')
+
+    registry_dictionary = {f'{PDB_ID}.{file_format}': None for PDB_ID in PDB_IDS}
+  
+    downloader = pooch.create(
+        path=cache_path,
+        base_url="https://files.wwpdb.org/download/",
```

Comment:
Can we make this a global module variable 
```python
#: Base URL for download files from the Protein Databank.
PDB_BASE_URL = "https://files.wwpdb.org/download/"
```
This will document what we're doing (also add `..autodata:: PDB_BASE_URL` to the docs) and show clearly where we need to change something if the URL changes.
```suggestion
        base_url=PDB_BASE_URL,
```

Reply:
Would this go something on [here](https://docs.mdanalysis.org/stable/documentation_pages/topology/PDBParser.html#module-MDAnalysis.topology.PDBParser)? Like under this would be under a new function with a new variable name. I'm unfamiliar in how sphinx builds docs.

Reply:
No, you can have a look at https://docs.mdanalysis.org/stable/documentation_pages/units.html#data to see how the docs are written. Add a new section in the docs at the top
```reST
Data
----
.. autodata:: PDB_BASE_URL

```
and then after the imports near the top of where global variables are typically defined
```python
#: Base URL for download files from the Protein Databank.
PDB_BASE_URL = "https://files.wwpdb.org/download/"
```

See sphinx docs for how `#: text` adds text as a doc string for a variable.

Reply:
I may have missed it but this seems unresolved?

Reply:
@BradyAJohnston See [https://github.com/MDAnalysis/mdanalysis/pull/4943#discussion_r2314132982](https://github.com/MDAnalysis/mdanalysis/pull/4943#discussion_r2314132982)

This PR has so many comments, it's a bit hard to find things

Reply:
There we some comments to _not_ make the URL a variable and I am ok with this alternative opinion so @jauy123 followed that discussion. Thus, this comment was properly resolved.